app/helpers/recruitment_banner_helper.rb

```diff
@@ -0,0 +1,21 @@
+module RecruitmentBannerHelper
+  def there_are_banners?
+    return false if banners.nil?
+
+    true
+  end
+
+  def renderable_banners
+    current_url = request.original_url
+
+    banners.select do |banner|
+      banner["page_urls"]&.include?(current_url)
+    end
+  end
+
+  def banners
```

Comment:
Banner is a bit of a loaded concept on govuk - there are emergency banners and global banners etc. So I think we could be more specific in our naming to avoid confusion... maybe `recruitment_banners` and apply this name to all the related methods you've created?